File: app/src/main/java/com/goranch/publicapis/api/model/food/nutrition/TotalHits.java
package com.goranch.publicapis.api.model.food.nutrition;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

public class TotalHits {

    @SerializedName("total_hits")
    @Expose
    private Integer totalHits;
    @SerializedName("max_score")
    @Expose
    private Double maxScore;
    @SerializedName("hits")
    @Expose
    private List<Hit> hits;

    public Integer getTotalHits() {
        return totalHits;
    }

    public void setTotalHits(Integer totalHits) {
        this.totalHits = totalHits;
    }

    public Double getMaxScore() {
        return maxScore;
    }

    public void setMaxScore(Double maxScore) {
        this.maxScore = maxScore;
    }

    public List<Hit> getHits() {
        return hits;
    }

    public void setHits(List<Hit> hits) {
        this.hits = hits;
    }

}

File: app/src/main/java/com/goranch/publicapis/api/model/food/nutrition/TotalHits.kt
package com.goranch.publicapis.api.model.food.nutrition

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class TotalHits {

    @SerializedName("total_hits")
    @Expose
    var totalHits: Int? = null
    @SerializedName("max_score")
    @Expose
    var maxScore: Double? = null
    @SerializedName("hits")
    @Expose
    var hits: List<Hit>? = null

}
